content/en/platform/corda/5.0-dev-preview-1/tutorials/building-cordapp/c5-basic-cordapp-flows.md

```diff
@@ -14,13 +14,15 @@ title: Write flows
 
 In Corda, flows automate the process of agreeing ledger updates. They are a sequence of steps that tell the node how to achieve a specific ledger update, such as issuing an asset or making a deposit. Nodes communicate using these flows in point-to-point interactions, rather than a global broadcast system. Network participants must specify what information needs to be sent, to which counterparties.
 
-Built-in flows are provided in Corda to automate common tasks, such as gathering signatures from counterparty nodes or notarising and recording a transaction. As a developer, you only need to implement flows for the business logic of your specific use case.
+Top-level flows encapsulate the business logic behind the interaction between the users and your CorDapp. They can have multiple subflows and can be started via RPC.
```

Comment:
not sure mentioning sub-flows here makes sense, it doesn't add to the information. Bring sub flows in when that "need" of the programmer crops up.

i.e. "need to do this in a flow" answe3r: Sub Flows

Reply:
It's just one simple sentence. Leaving as is.